sdk/python/kfp/client/client.py

```diff
@@ -1020,6 +1020,7 @@ def create_run_from_pipeline_func(
         pipeline_func: base_component.BaseComponent,
         arguments: Optional[Dict[str, Any]] = None,
         run_name: Optional[str] = None,
+        experiment_id: Optional[str] = None,
```

Comment:
New parameters should be added to the end to not break existing users who might call the method using positional arguments.

Reply:
Good call. Didn't think of that! Resolved.